config/jobs/etcd/etcd-presubmits.yaml

```diff
@@ -447,3 +447,32 @@ presubmits:
             privileged: true
       nodeSelector:
         kubernetes.io/arch: arm64
+
+  etcd-io/website:
```

Comment:
Would it make sense to put website jobs in their own file?  cc @serathius, @ahrtr for any opinion on this.

Think I would prefer to have each repository have it's own files rather than one big monolithic file.

Reply:
I initially had it in a different file, which I feel is better to avoid having a gigantic file. However, I looked into other projects' configuration files, and having them in the same file seems more standard.

So, if you were going to ask me, my vote would be the same as yours.

Reply:
> Would it make sense to put website jobs in their own file? cc @serathius, @ahrtr for any opinion on this.

I do not have strong opinion on this. Either way works for me. thx

Reply:
With two votes on a new file. I decided to update the PR by adding a new file.